swap_meet/vendor.py

```diff
@@ -1,2 +1,59 @@
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+
+        return False
```

Comment:
I like reversing the sense of the condition to be able to treat it like a guard clause, which allows us to outdent the main logic of the function something like the following:
```python
        if item not in self.inventory:
            return False

        self.inventory.remove(item)
        return item
```

Another approach we could take is to try to remove the item directly, and handle the `ValueError` that occurs if it's not there, and return `False` to handle it.